application.go

```diff
@@ -0,0 +1,144 @@
+// Package mesg is an application client for mesg-core.
+// For more information please visit https://mesg.com.
+package mesg
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/mesg-foundation/core/api/core"
+	"google.golang.org/grpc"
+)
+
+const (
+	endpointEnv     = "MESG_ENDPOINT"
```

Comment:
Yes, we also have a configurable endpoint from env for application in [mesg-js](https://github.com/mesg-foundation/mesg-js/blob/master/src/index.ts#L9). I tought we'll continue to support env vars for endpoint modification but in the current implementation in go-application this also can be done with Options while creating a New Application. Should we totally drop support for env vars for setting a custom endpoint at this point?

Reply:
About `mesg-js` it's because the lib is for application and service and I think we leave the env for application by mistake.

I would definitely drop support for ENV in the application in favor of the Option. It's up to the dev of the application to create it's own ENV variable on its host machine and pass it to the lib.

@antho1404 do you confirm?

Reply:
Yes agree, the `MESG_ENDPOINT` is only injected for the services. For the application it's up to the developer to set it's own env variable.